### ISSUE TITLE
Reverse image search error

### ISSUE THREAD

**Issue by stephenhawk8054** [libredirect/libredirect#91]
Hi, I was testing google's reverse image search, and I think both SearX and Whoogle cannot redirect `searchbyimage`.

For example, with this link:

```
https://www.google.com/searchbyimage?image_url=https%3A%2F%2Fwikiless.org%2Fmedia%2Fwikipedia%2Fcommons%2F0%2F0d%2FScreenshot_of_the_uBlock_Origin_UI_on_Firefox.png
```

Both SearX and Whoogle instances return error page or redirect to home page. Would it be possible to exclude `searchbyimage` in `targets` regex?

**Comment by ManeraKai** [comment]
Does SearX or Whoogle support searching by image in the first place?

**Comment by stephenhawk8054** [comment]
I don't think so. Maybe I haven't looked into the preferences well enough but as far as I understand, they just support searching images by given texts, but not searching images similar to a given image (or reverse image search function in google, bing, yandex... I don't know how to describe it correctly).

**Comment by ManeraKai** [comment]
I'll be happy to support this feature once they added it, For now, I'll close this issue.

**Comment by stephenhawk8054** [comment]
> I'll be happy to support this feature once they add it, For now, I'll close this issue.

Then I think we should exclude `searchbyimage` to not be redirected to searx or whoogle in the regex right? Because right now accessing that link will give error in both instances types.

**Comment by ManeraKai** [comment]
You're right. Will do that this day.

**Comment by stephenhawk8054** [comment]
Oh I see. I didn't realize it redirects to another query. Tks for your quick fix 👍
